scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/server/RestClient.java

```diff
@@ -785,10 +786,17 @@ void putThirdPartyCredential(@HeaderParam("sessionid") String sessionId, @PathPa
     @Produces(MediaType.APPLICATION_JSON)
     Map<Object, Object> getSchedulerPortalDisplayProperties(@HeaderParam("sessionid") String sessionId);
 
-    @GET
+   @GET
     @Path("job/{jobid}/permission/{method}")
     @Produces(MediaType.APPLICATION_JSON)
     InputStream checkJobPermissionMethod(@HeaderParam("sessionid") String sessionId, @PathParam("jobid") String jobId,
             @PathParam("method") String method);
+  
+    @POST
+    @Path("job/{jobid}/signals")
+    @Consumes(value = MediaType.APPLICATION_JSON)
+    @Produces("application/json")
+    Set<String> addJobSignal(@HeaderParam("sessionid") String sessionId, @QueryParam("signal") String signal,
```

Comment:
Set<String> ?

Reply:
still waiting for an answer

Reply:
Sorry, the request returns the list of the job's signals after the 'add signal' operation